### PR TITLE
Fix delete disk

### PR DESCRIPTION
**Pull request by ebattat**
## Type of change
Note: Fill **x** in []
- [x] bug
- [ ] enhancement
- [ ] documentation
- [ ] dependencies

## Description

Got the following error while ODF installation
`Message:      failed to configure devices: failed to get device already provisioned by ceph-volume raw: osd.8:`
Fix delete ODF/ Ceph disks according to [recommendation](https://rook.io/docs/rook/v1.12/Getting-Started/ceph-teardown/#delete-the-data-on-hosts)

### CODE REVIEW

File: benchmark_runner/common/ocp_resources/create_ocp_resource.py [hunk 1]
```diff
@@ -77,23 +77,23 @@ def create_resource(self, resource: str):
         self.remove_resource_files(path=os.path.join(self.__dir_path, resource))
         resource_files = self.get_sorted_resources(resource=resource)
         if 'lso' == resource:
-            self.__create_lso = CreateLSO(self.__oc, path=os.path.join(self.__dir_path, resource), resource_list=resource_files)
-            self.__create_lso.create_lso()
+            create_lso = CreateLSO(self.__oc, path=os.path.join(self.__dir_path, resource), resource_list=resource_files)
+            create_lso.create_lso()
```
Comment (RobertKrawitz) [inline benchmark_runner/common/ocp_resources/create_ocp_resource.py:81]: What's the issue with this?

File: benchmark_runner/common/ocp_resources/odf/template/01_delete_disks_template.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+while (($# >= 2)) ; do
+    node=$1
+    delete_disk_command=$2
```
Comment (RobertKrawitz) [inline benchmark_runner/common/ocp_resources/odf/template/01_delete_disks_template.sh:5]: Should be "=$2" (in quotes) so the shell won't process the value of $2.

Comment (ebattat) [reply benchmark_runner/common/ocp_resources/odf/template/01_delete_disks_template.sh]: Fixed

File: benchmark_runner/common/ocp_resources/create_ocp_resource.py (hunk 1 above)
Comment (ebattat) [reply benchmark_runner/common/ocp_resources/create_ocp_resource.py:81]: No need to create class object if you are not using it in several places

**Pull request merged**